docs/config.rst

```diff
@@ -12,6 +12,8 @@ Configuration Options
 ``listeners``
   Comma-separated list of listeners that listen for API requests over either HTTP or HTTPS. If a listener uses HTTPS, the appropriate SSL configuration parameters need to be set as well.
 
+  Schema Registry identities are stored in ZooKeeper and are made up of a hostname and port. If multiple listeners are configured, the first listener's port is used for its identity.
```

Comment:
Hmm, that seems a bit excessive. Its not clear every rest-utils project would do this. My concern is just that this will get overwritten when we regenerate this file.

`listeners` is specified with a `defineOverride` in the config class. Can we just make the doc string the old one + this extra string?